include/deal.II/grid/reference_cell.h

```diff
@@ -2610,6 +2612,26 @@ ReferenceCell::standard_vs_true_line_orientation(
       return (line_orientation ==
               bool_table[line / 2][combined_face_orientation]);
     }
+  else if (*this == ReferenceCells::Tetrahedron)
+    {
+      static constexpr unsigned int X = numbers::invalid_unsigned_int;
+      static constexpr dealii::ndarray<unsigned int, 4, 3> combined_lines{
+        {{{0, 0, 0}}, {{X, 0, 1}}, {{X, 0, X}}, {{X, X, X}}}};
+
+      const auto combined_line = combined_lines[face][line];
+
+      Assert(combined_line != X,
+             ExcMessage(
+               "This function can only be called for following face-line "
+               "combinations: (0,0), (0,1), (0,2), (1,1), (1,2), (2,1),"));
```

Comment:
I think this restriction is fine since this is not a function users would be calling.